test-app-reactive/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstemplatepackagenameasync/service/MessageListener.kt

```diff
@@ -13,11 +13,12 @@ import java.util.concurrent.CompletableFuture
 class InboundMessageListener(private val inboundMessageService: InboundMessageService, private val objectMapper: ObjectMapper) {
 
   @SqsListener("inboundqueue", factory = "hmppsQueueContainerFactoryProxy")
-  fun processMessage(message: Message): CompletableFuture<Void> {
+  fun processMessage(message: Message): CompletableFuture<Void?> {
     val event = objectMapper.readValue(message.Message, HmppsEvent::class.java)
     return CoroutineScope(Context.current().asContextElement()).future {
       inboundMessageService.handleMessage(event)
-    }.thenAccept {}
```

Comment:
avoid use of `thenAccept` as it encourages people to put code in here which doesn't then seem to work with span tracking